backend/algorithms/errors.json

```diff
@@ -11829,14 +11278,16 @@
     "GMAT2130": {
         "condition": {
             "original": "Prerequisite: GMAT1110, Corequisite: GMAT2120, GMAT2500, GMAT2700.<br/><br/>",
-            "processed": "GMAT1110, [GMAT2120, GMAT2500, GMAT2700]"
+            "processed": "GMAT1110, [GMAT2120 && GMAT2500 && GMAT2700]"
```

Comment:
Not intended at all, should have a check to make sure we're not in a coreq bracket